eng/Versions.props

```diff
@@ -5,7 +5,7 @@
 
     <!-- This repo version -->
     <VersionPrefix>1.0.0</VersionPrefix>
-    <PreReleaseVersionLabel>beta</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>beta2</PreReleaseVersionLabel>
```

Comment:
Is `PreReleaseVersionLabel` a new property?

Reply:
It's property defined by Arcade.